### PR TITLE
temporarily add deepCore and displacedRegional workflows to runTheMatrix

### PR DESCRIPTION
**Pull request by slava77**
deepCore and displacedRegional tracking updates are planned for full incremental validation in 14_0 and once validated, will be proposed for production.

In view of this, to improve quality assurance this PR is proposed to add two test workflows in standard PR tests.
Up to several weeks should be enough to check behavior in non-standard builds as well as reproducibility.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
> I found 1 errors in the following unit tests:

```
runTheMatrix.py", line 101
    14034.0, #2023 ttbar fastsim
    ^
SyntaxError: invalid syntax
```

sorry, was my fault, making a typo in the runTheMatrix (I missed a comma in the list).

@iarspider @smuzaffar 
would it be possible to detect issues like this in the code-checks?

**Comment by slava77** [comment]
@cmsbuild please test

File: Configuration/PyReleaseValidation/scripts/runTheMatrix.py [hunk 1]
```diff
@@ -92,10 +92,12 @@ def runSelected(opt):
                      11634.911, #2021 DD4hep ttbar reading geometry from XML
                      11634.914, #2021 DDD ttbar reading geometry from the DB
                      11634.0, #2021 ttbar (switching to DD4hep by default)
+                     11723.17, #2021 QCD Pt_1800_2400 seedingDeepCore; Temporary
                      13234.0, #2021 ttbar fastsim
                      12434.0, #2023 ttbar
                      12634.0, #2023 ttbar PU
                      12434.7, #2023 ttbar mkFit
+                     12434.701, #2023 ttbar displacedRegional; Temporary
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/scripts/runTheMatrix.py:100]: IIUC, in order to make this available by default (i.e. without specifying `--what upgrade`) in the limited matrix it needs to appear in this list as well:

https://github.com/cms-sw/cmssw/blob/27bec5ff620bde58ac4a2a2812a545e881c47117/Configuration/PyReleaseValidation/python/relval_2017.py#L50

Comment (srimanob) [reply Configuration/PyReleaseValidation/scripts/runTheMatrix.py:100]: That is correct. Please add it to the list.

**Comment by slava77** [comment]
following the RECO meeting, this PR may be not useful after all
there is a separate PR proposing to enable both deepCore and displacedRegional: #43472 
So, I will close this for now

@cms-sw/reconstruction-l2

**Pull request closed without merging**